TOOLS/lua/autoload.lua

```diff
@@ -22,6 +28,7 @@ additional_audio_exts=list,of,ext
 ignore_hidden=yes
 same_type=yes
 directory_mode=recursive
+ignore_patterns=^~,^bak-,%.bak%.mp4$
```

Comment:
```suggestion
ignore_patterns=^~,^bak-,%.bak$
```
I would make the default more generic. `mp4` is quite specific and why not add all other media extensions? So I would just remove and leave `.bak`